website/docs/r/compute_fleet.html.markdown

```diff
@@ -0,0 +1,740 @@
+
+* `zones` - (Optional) Specifies a list of availability zones in which the Compute Fleet is available. Changing this forces a new resource to be created.
+
+* `compute_api_version` - (Optional) Specifies the `Microsoft.Compute` API version to use when creating the Compute Fleet. Changing this forces a new resource to be created.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `regular_priority_profile` - (Optional) A `regular_priority_profile` block as defined below.
+
+* `spot_priority_profile` - (Optional) A `spot_priority_profile` block as defined below.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Compute Fleet.
+
+* `vm_sizes_profile` - (Optional) One or more `vm_sizes_profile` blocks as defined below.
+
+-> **Note:** If `spot_priority_profile` is specified, `regular_priority_profile` is not specified and `spot_priority_profile.0.maintain_enabled` is specified as to `false`, changing `vm_sizes_profile` forces a new resource to be created.
+
+---
+
+A `virtual_machine_profile` block supports the following:
+
+* `network_api_version` - (Required) Specifies the Microsoft.Network API version used when creating networking resources in the network interface configurations for the Compute Fleet. Changing this forces a new resource to be created.
+
+* `network_interface` - (Required) One or more `network_interface` blocks as defined above. Changing this forces a new resource to be created.
+
+* `os_profile` - (Required) A `os_profile` block as defined above. Changing this forces a new resource to be created.
+
+* `boot_diagnostic_enabled` - (Optional) Whether to enable the boot diagnostics on the virtual machine. Defaults to `false`. Changing this forces a new resource to be created.
+
+* `boot_diagnostic_storage_account_endpoint` - (Optional) Specifies endpoint of the storage account to use for placing the console output and screenshot. Changing this forces a new resource to be created.
+
+* `capacity_reservation_group_id` - (Optional) Specifies the ID of the capacity reservation group which the Compute Fleet should be allocated to. Changing this forces a new resource to be created.
+
+* `data_disk` - (Optional) One or more `data_disk` blocks as defined above. Changing this forces a new resource to be created.
+
+* `encryption_at_host_enabled` - (Optional) Whether to enable encryption at host on disks attached to the Compute Fleet. Defaults to `false`. Changing this forces a new resource to be created.
+
+* `extension` - (Optional) One or more `extension` blocks as defined above. Changing this forces a new resource to be created.
+
+* `extension_operations_enabled` - (Optional) Whether to enable extension operations on the Compute Fleet.  Defaults to `true`. Changing this forces a new resource to be created.
+
+* `extensions_time_budget` - (Optional) Specifies the time alloted for all extensions to start. Changing this forces a new resource to be created.
```

Comment:
what unit is this in? seconds? min? duration?

Reply:
The unit should be duration, fixed.